movie-browser/src/App.js

```diff
@@ -21,13 +24,27 @@ class App extends Component {
       });
   }
 
+  handleToggleMovies = () => {
+    if (!this.state.toggleMovies) {
```

Comment:
```suggestion
   const { toggleMovies } = this.state;
```

Reply:
Will always destructure going forward

Reply:
alright. it's just a minor thing. Not a blocker.